ecosystem/sep-0024.md

```diff
@@ -190,6 +191,72 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 
+##### Claimable Balances
+
+[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) allows payments to accounts that do not have a trust-line for the asset sent.
+
+**Wallets** poll for their users' received Claimable Balances so that users can "claim" them.
+
+**Anchors** can optionally use this functionality to send SEP-24 deposit transactions to Stellar accounts that don't yet have a trust-line to the asset they have requested.
+
+**Wallet Claimable Balance Flow**
+
+Wallets will still follow the [typical deposit flow](#Basic-Wallet-Implementation).
+The POST request however will extend to include a `claimable_balance` parameter
+
+```
+POST TRANSFER_SERVER_SEP0024/transactions/deposit/interactive
+Content-Type: multipart/form-data
+```
+
+New Request Parameter:
+
+Name | Type | Description
+-----|------|------------
+`claimable_balance` | boolean | True if submitting transactions as a claimable balance is accepted, false otherwise.
+
+**Wallet Claimable Balance Polling**
+
+The wallet must either listen for a callback from the anchor with the
+`claimable_balance_id` or poll Horizon's
+[`/claimable_balances/`](https://developers.stellar.org/api/resources/claimablebalances/list/)
+endpoint to pull any available claimable balances for the claimant.
+
+*Recommended request*
+`GET /claimable_balances?claimant={:claimant}&order={asc,desc}&limit={1-200}`
+
+This allows the wallet to update the claimant that their is a balance available
+to claim and they may need to establish a [trust-line](#stellar-account-doesnt-trust-asset) in order to do so.
+
+**Anchor Claimable Balance Flow**
+
+1. The given Stellar `account` does not have a trust-line to the asset's issuer account.
+2. The `Anchor` proceeds with the deposit flow via the [CAP-0023](../core/cap-0023.md) Create Claimable Balance operation.
+
+**Anchor Claimable Balance Field Changes**
+
+### Fields for deposit transactions
+
+Name | Type | Description
+-----|------|------------
+`claimable_balance_id` | string | (optional) ClaimableBalanceID is a required parameter when claiming a Claimable Balance operation.
+
+The `claimable_balance_id` can be added to the [response](#deposit-and-withdraw-shared-responses)
+This can be helpful for wallets to communicate to the user that their
+transaction was submitted as a claimable balance and can be claimed at the relevant location (`claimable_balance_id`).
+
+In order to claim it the recipient must either get access to the account that the anchor created and or establish a trust-line (for the asset code of the claimableBalance) with their existing account.
+
+**Anchor Claimable Balance Recommended Options**
+
+#### Claimable Balance Predicates
+
+[https://developers.stellar.org/docs/glossary/claimable-balance/#parameters](https://developers.stellar.org/docs/glossary/claimable-balance/#parameters)
+
+We suggest if you decide to set predicates to claim the balance in 30 days. `BEFORE_RELATIVE_TIME(X)`
```

Comment:
Did we decide 30 days was a good suggestion? Whats the rationale?

Reply:
We discussed that 30 days might be a good sweet spot to not keep balances piling up on the dex without them being claimed.

I'm still open to discussion on this one.

On one hand I think its totally fine to keep in unconditional and let the recipient claim on their own time whether it be a week a month or a year.
On the other hand, I think that if we help push recipients to claim their balance with a bit of haste it could lead them to do more activity (payments and whatnot) on the network. Instead of having balances eternally rot.